src/vcModals.cpp

```diff
@@ -632,16 +631,31 @@ void vcModals_DrawProjectInfo(vcState *pProgramState)
     else
       pProgramState->modalOpen = true;
 
-    const char *pInfo = nullptr;
-
     if (ImGui::BeginChild("##infoModal", ImVec2(-1, -30), true))
     {
-      if (udProjectNode_GetMetadataString(pProgramState->activeProject.pRoot, "information", &pInfo, "") == udE_Success)
-        ImGui::TextWrapped("%s", pInfo);
+      for (size_t i = 0; i < pProgramState->projectInfoTextures.infoStrings.length; ++i)
+      {
+        ImGui::TextWrapped("%s", pProgramState->projectInfoTextures.infoStrings[i]);
+        if (i < pProgramState->projectInfoTextures.infoStrings.length - 1)
+        {
+          if (*pProgramState->projectInfoTextures.textures[i] == nullptr)
```

Comment:
This check is unnecessary, `vcTexture_GetSize` will return `udR_InvalidParameter` when the texture is `nullptr`.